flaskr_tests.py

```diff
@@ -133,6 +121,71 @@ def test_move(self):
         assert '1' in id_order[0]
         assert '2' in id_order[1]
 
+    def test_unittest_move_entry(self):
+        """Test entries are moved in the database."""
+        with flaskr.app.app_context():
+            flaskr.init_db()
+            db = flaskr.get_db()
+            self.test_move()
```

Comment:
Since this is "unittest", you need to directly use lower level (db-level) calls to move the record, so you need to extract the "unit" from the existing code first (into separate class of function) and then use it, here, in the test, as well as in the route (https://github.com/al-serebrov/flaskr/blob/master/flaskr.py#L179).